requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/Labelbox/vertex-model-training.git#egg=source_code&subdirectory=source_code
```

Comment:
Same comment.. This shoudl be directly referenced in the subfolders from the root main.py.  Here we should only have Labelbox SDK and GCP SDK direclty referenced.